gn3/computations/correlation_matrix.py

```diff
@@ -16,36 +16,15 @@
 from gn3.computations.correlations import normalize_values
 
 
-def compute_the_pca(data, transform: bool = True):
-    """function to compute pca"""
-    pca = PCA()
-    if transform is True:
-        scaled_data = preprocessing.scale(data.T)
-
-    else:
-        scaled_data = preprocessing.scale(data)
-
-    # compute loading score and variaton of each pc
-
-    # generate coordinates for pca graph if needd
-    pca.fit(scaled_data)
-
-    pca_data = pca.transform(scaled_data)
-
-    return (pca, pca_data)
-
-
-def compute_pca2(matrix):
-    """compute the pca"""
+def compute_pca(matrix):
+    """compute principal component analyis"""
 
     pca = PCA()
+
     scaled_data = preprocessing.scale(matrix)
     pca.fit(scaled_data)
-
+    # generates coordinated based on the loading_scores and scaled data
```

Comment:
**nitpick:** Generally make comments in the active voice. e.g.: "Generate co-ordinates based on the loading..."